ygen/testdata/structs/openconfig-list-enum-key.getters-append.formatted-txt

```diff
@@ -150,6 +177,21 @@ func (t *OpenconfigListEnumKey_Top_MultiKey) GetOrCreateEkm(K1 E_OpenconfigListE
 	return v
 }
 
+// GetEkm retrieves the value with the specified key from
+// the Ekm map field of OpenconfigListEnumKey_Top_MultiKey.
+func (t *OpenconfigListEnumKey_Top_MultiKey) GetEkm(K1 E_OpenconfigListEnumKey_Top_MultiKey_Ekm_Config_K1, K2 E_OpenconfigListEnumKey_FooIdentity) (*OpenconfigListEnumKey_Top_MultiKey_Ekm){
+
+  key := OpenconfigListEnumKey_Top_MultiKey_Ekm_Key{
+		K1: K1,
+		K2: K2,
+	}
+
+  if lm, ok := t.Ekm[key]; ok {
```

Comment:
Similarly here, looks like this wouldn't handle a nil receiver.

Reply:
Fixed :-)